096-drop-travis-ci-and-testing-for-ppc-and-s390x.md

```diff
@@ -0,0 +1,63 @@
+# Drop Travis-CI and testing efforts for `ppc64le` and `s390x` architectures
+
+This proposal suggests to drop current usage of [Travis CI](https://www.travis-ci.com/) in [strimzi-kafka-bridge](https://github.com/strimzi/strimzi-kafka-bridge) and [strimzi-kafka-oauth](https://github.com/strimzi/strimzi-kafka-oauth) repositories.
+
+## Motivation
+
+In the past, we adopted Azure Pipelines as the primary CI system for the Operators repository, using it to run tests, build documentation, and create artifacts and images.
+The main reason for this transition was the improved resource quotas available for CNCF projects, which allowed us to run pipelines with far fewer restrictions than we had in Travis CI. 
+However, Azure Pipelines does not provide agents for the `ppc64le` and `s390x` architectures.
+This was not an issue for Operators, as we build multi-architecture images on `amd64` using _docker buildx_. 
+However, for Strimzi Kafka Bridge and Strimzi Kafka OAuth, we decided to retain the testing pipelines for `ppc64le` and `s390x`.
+
+In recent months, we have encountered several unexpected issues with Travis CI that required us to submit support tickets to restore our quota.
+These issues are often not immediately noticed when they occur but are only discovered weeks later. 
+It seems that no one even realizes that some pipelines fail to execute from time to time.
+Another scenario is that jobs fail due to various Travis CI issues, yet we simply ignore the results and merge the PRs anyway.
```

Comment:
The CI issues were spurious or could not be determined?  or not worth the time to raise?

Reply:
I think we just do not realize we have a problems for some time as Travis is not our main CI tool we use and it would be much better for us to use just one CI tool if possible. Travis kinda adds additional work that we would like to avoid.